src/main/resources/config/oauth/security.yml

```
@@ -31,3 +31,4 @@ oauth:
       - "https://ssok.info"
       - "https://api.ssok.info"
       - "https://dev-api.ssok.info"
+logging.level.org.springframework.security: DEBUG
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)